ansible/group_vars/nodepool.yaml

```diff
@@ -33,21 +33,26 @@ nodepool_service_nodepool_launcher_enabled: false
 nodepool_service_nodepool_launcher_manage: false
 nodepool_service_nodepool_launcher_state: stopped
 
+nodepool_user_name: nodepool
+nodepool_user_group: nodepool
+nodepool_user_home: /var/lib/nodepool
+
 nodepool_pip_version: 3.12.0
 nodepool_pip_virtualenv_python: python3
 nodepool_pip_virtualenv: "/opt/venv/nodepool-{{ nodepool_pip_version }}"
 
 # windmill.openstacksdk
-openstacksdk_user_name: nodepool
```

Comment:
this won't work, as nodepool_user_name isn't loaded at this point in time. So we hardcode to nodepool